public/locales/de/app.json

```
@@ -86,6 +86,8 @@
   "payment_type.prepayment": "Prepayment",
   "payment_type.wechatpay": "WeChat Pay",
   "payment_type.banktransfer": "Bank Transfer",
+  "payment_type.openbanking-pis": "Open Banking",
+  "payment_type.clicktopay": "Click2Pay",
```

Please use the official product name for clicktopay
```suggestion
  "payment_type.clicktopay": "Click to Pay",
```